### PR TITLE
Align FR with EN.

### PR DESCRIPTION
**Pull request by jeberhardt**
# Summary | Résumé

Align the FR copy with the EN copy such that each entry is on the same line, for easier editing.

# Test instructions | Instructions pour tester la modification

Compare the EN and FR files and ensure that each entry is on the same line.

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

This PR should be merged before any other changes to copy occur.

### CODE REVIEW

**Comment by timarney** [comment]
@aklef can you do a local build to test this?

File: src/locale/translations/fr.json [hunk 1]
```diff
@@ -214,7 +53,34 @@
...
     },
+    "ChooseRegionCTA": "Sélectionnez une province ou un territoire",
+    "How": "Diagnosed with COVID-19?",
+    "CTA": "Prochaines étapes",
+    "HowUrl": "https://www.canada.ca/",
     "ExposedHelpCTA": "Que faire en cas d’exposition",
+    "DiagnosedView": {
+      "Title": "Merci d’aider à lutter contre la COVID",
+      "Body1": {
+        "One": "Tomorrow, you’ll get your last notification asking permission to upload any new random codes your phone sent out.",
```
Comment (aklef) [inline src/locale/translations/fr.json:64]: This needs to be translated to french

**Review by aklef — COMMENTED** [review]
I compared the EN and FR JSON files and ensured that each entry from one file is on the corresponding line in the french file. Waiting to see if the missing french translation is significant enough to block the merge

**Comment by jeberhardt** [comment]
> Waiting to see if the missing french translation is significant enough to block the merge

I don't think it should block the merge if the EN translation existed in the FR file before this PR.

File: src/locale/translations/fr.json (hunk 1 above)
Comment (timarney) [reply src/locale/translations/fr.json:64]: Checked to see if this exists in the code and it does.

```
export function pluralizeKey(baseKey: string, number: number) {
  if (number === 1) return `${baseKey}.One`;
  return `${baseKey}.Other`;
}
```

**Review by timarney — APPROVED** [review]

**Pull request merged**